final_task/rss_reader/json_structure.txt

```diff
@@ -0,0 +1,20 @@
+JSON structure:
+
+{
+  "feed": { "title": feed title,
+            "date":  feed publication date,
+            "language": feed language }
+  "entries": [
+               { "title": news title,
+                 "summary": news content,
+                 "date": news publication date,
+                 "link": news link },
+
+               { "title": news title,
+                 "summary": news content,
+                 "date": news publication date,
+                 "link": news link },
+
+               ...
+             ]
+}
```

Comment:
It is better to move it into `final_task/README.md`. All technical info is supposed to be there.

Reply:
fixed